staging/src/k8s.io/cli-runtime/pkg/resource/dry_run_verifier.go

```diff
@@ -29,16 +29,23 @@ import (
 
 // VerifyDryRun returns nil if a resource group-version-kind supports
 // server-side dry-run. Otherwise, an error is returned.
-func VerifyDryRun(gvk schema.GroupVersionKind, dynamicClient dynamic.Interface, discoveryClient discovery.DiscoveryInterface) error {
-	verifier := NewDryRunVerifier(dynamicClient, discoveryClient)
+func VerifyDryRun(gvk schema.GroupVersionKind, dynamicClient dynamic.Interface, openAPIDoc *openapi_v2.Document) error {
```

Comment:
I think this function is never called. All of kubectl cmd use NewDryRunVerifier and (DryRunVerifier).HasSupport method to verify dryrun

Reply:
That's a good point. We should probably remove it if it's not used. I think it used to, but the call site must have been refactored, thanks!

Reply:
Removed